tests/verilog/genvar_loop_decl_1.ys

```diff
@@ -0,0 +1,14 @@
+read_verilog -sv genvar_loop_decl_1.sv
+
+select -assert-count 1 gate/genblk1[0].x
+select -assert-count 1 gate/genblk1[1].x
+select -assert-count 0 gate/genblk1[2].x
+
+select -assert-count 1 gold/genblk1[0].x
+select -assert-count 1 gold/genblk1[1].x
+select -assert-count 0 gold/genblk1[2].x
+
+proc
+equiv_make gold gate equiv
```

Comment:
This equivalence check doesn't actually do anything here, since the x wires are undriven — the equiv still passes if eg. I have wrong widths for `x` in some iteration